dfastmi/batch/core.py

```diff
@@ -245,26 +245,41 @@ def _report_analysis_conditions_header(report: TextIO):
     ApplicationSettingsHelper.log_text("analysis_settings_conditions_header", report)
 
 
-def _report_used_file_names(config: ConfigParser, q_threshold: float, report: TextIO):
-    for section in config:
-        if section[0].lower() == "c" or section[0].lower() == "q":
-            discharge_str = config.get(section, "Discharge", fallback="")
-            discharge = float(discharge_str)
-            condition = "{:7.1f} m3/s".format(discharge)
-            if discharge <= q_threshold:
-                _report_analysis_conditions_values(
-                    condition, "---", "---     (measure not active)", report
-                )
-            else:
-                reference_file_name = _get_file_name(config, section, "Reference")
-                measure_file_name = _get_file_name(config, section, "WithMeasure")
-                _report_analysis_conditions_values(
-                    condition, reference_file_name, measure_file_name, report
-                )
+def _report_used_file_names(
+    config: ConfigParser,
+    initialized_config: AConfigurationInitializerBase,
+    report: TextIO,
+):
+    imode = _get_mode_usage(config)
+    filenames = get_filenames(imode, initialized_config.needs_tide, config)
+    for i, q in enumerate(initialized_config.discharges):
+        if not q:  # should only happen for version 1 files
+            continue
+        if 0 in filenames:
+            key = i
+        elif initialized_config.needs_tide:
+            key = (q, initialized_config.tide_bc[i])
+        else:
+            key = q
+        condition = "{:7.1f} m3/s".format(q)
```

Comment:
Suggestion to move this to a descriptable method and refactor a litte bit, see example:
```
        condition, reference_file_name, measure_file_name, comment = get_analysis_condition_values_for_logging(initialized_config, filenames, q, key)

        _report_analysis_conditions_values(
            condition, reference_file_name, measure_file_name, comment, report
        )


def get_analysis_condition_values_for_logging(initialized_config, filenames, q, key):
    condition = "{:7.1f} m3/s".format(q)
    
    if q <= initialized_config.q_threshold:
        reference_file_name = "---"
        measure_file_name = "---"
        comment = "(measure not active)"
    elif key in filenames:
        files = filenames[key]
        reference_file_name = _get_file_name(files[0])
        measure_file_name = _get_file_name(files[1])
        comment = ""
    else:
        reference_file_name = "xxx"
        measure_file_name = "xxx"
        comment = "(not specified)"
        
    return condition, reference_file_name, measure_file_name, comment
```

This example moves the behviour that decides the data for logging to a method.
The actual message in the UK and NL messages files should also be updated to include comment, as the _report_analysis_conditions_values can have the comment inserted seperately and wont have it merged into the measure message.

Reply:
Resolved see PR #102

Reply:
Discussed with Bert.